packages/react-dom/src/__tests__/ReactCompositeComponent-test.js

```diff
@@ -432,8 +432,8 @@ describe('ReactCompositeComponent', () => {
     }).toWarnDev(
       'Cannot update during an existing state transition (such as within ' +
         "`render` or another component's constructor). Render methods should " +
-        'be a pure function of props and state; constructor side-effects are ' +
-        'an anti-pattern, but can be moved to `componentWillMount`.',
+        'be a pure function of props and state. ' +
+        'Move the setState call to componentDidMount`.',
```

Comment:
This is too prescriptive. We don't actually *know* they called setState in constructor, or that they want to move it to cDM (setting it in constructor might be better in most cases).

Reply:
@gaearon I agree:) Whatever, i thought deprecated lifecycle error message should be deleted! 
As you say, I will fix pr by setting warn message by deleting message about constructor and cDM

Reply:
@gaearon is it ok?